.github/workflows/build-and-test.yml

```
@@ -59,6 +59,16 @@ jobs:
         restore-keys: |
           cargo-build-target-${{ hashFiles('**/Cargo.toml') }}
 
+    - name: Cache nodejs binding cargo target
+      uses: actions/cache@v2
+      with:
+        path: bindings/node/native/target
+        # Add date to the cache to keep it up to date
+        key: cargo-build-target-${{ hashFiles('**/Cargo.toml') }}-${{ env.CURRENT_DATE }}
```

We should use a different cache key here so it doesn't conflict with the one above

```suggestion
        key: cargo-build-node-target-${{ hashFiles('**/Cargo.toml') }}-${{ env.CURRENT_DATE }}
```